linera-service/tests/fixtures/evm_example_process_streams.sol

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+import "./Linera.sol";
+import "./LineraTypes.sol";
+
+contract ExampleExecuteMessage {
+    uint64 total_value;
+    mapping(bytes32 => uint64) chain_values;
+
+    function subscribe(bytes32 chain_id1, bytes32 application_id) external {
+        bytes memory stream_name = abi.encodePacked(uint8(42));
+        Linera.subscribe_to_events(chain_id1, application_id, stream_name);
+    }
+
+    function increment_value(uint64 increment) external {
+        total_value = total_value + increment;
+        bytes memory stream_name = abi.encodePacked(uint8(42));
+        bytes memory value = abi.encode(total_value);
+        Linera.linera_emit(stream_name, value);
+    }
+
+    function process_streams(bytes memory input) external {
+        LineraTypes.StreamUpdates memory updates = LineraTypes.bcs_deserialize_StreamUpdates(input);
+        bytes memory stream_name = abi.encodePacked(uint8(42));
```

Comment:
Extract to a [constant](https://solidity-by-example.org/constants/)

Reply:
You are right about that.
However, we cannot put a constant value that is a bytes. So, the stream_name cannot be put as such a constant.